packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -102,12 +102,13 @@ export default class IconButtonCore extends React.Component<Props> {
         };
 
         if (href) {
+            const actualHref = !disabled ? href : undefined;
             return clientNav ? (
-                <StyledLink {...commonProps} to={href}>
+                <StyledLink {...commonProps} to={actualHref}>
                     {child}
                 </StyledLink>
             ) : (
-                <StyledAnchor {...commonProps} href={href}>
+                <StyledAnchor {...commonProps} href={actualHref}>
```

Comment:
🛑 You're doing this in a few places now and I'm not sure this is right... we'll end up with `<a>`s that have no `href` - which doesn't seem to be the intended effect? An anchor without a `href` behaves very differently (as it's meant to represent an anchor point in a page to jump to). If we don't want the link to do anything when clicked we either need to do `preventDefault` (which is what I think we had before) or turn the link into something else like `javascript: void 0`

Reply:
I've change this to use `preventDefault` but on for anchors.

Reply:
> An anchor without a href behaves very differently (as it's meant to represent an anchor point in a page to jump to)

The main change I saw in behaviour was that the cursor doesn't change to the "pointer" on mouseover.  What are the other changes?